src/modules/deal-guarantee/dto/create-deal-guarantee-request.dto.ts

```diff
@@ -9,9 +9,10 @@ export type CreateDealGuaranteeRequest = CreateDealGuaranteeRequestItem[];
 export class CreateDealGuaranteeRequestItem {
   @ValidatedStringApiProperty({
     description: 'The identifier of the deal to create the guarantee for.',
-    example: '00000001',
-    minLength: 8,
-    maxLength: 8,
+    example: EXAMPLES.DEAL_ID,
+    minLength: 10,
+    maxLength: 10,
+    pattern: /^00\d{8}$/,
```

Comment:
Any chance we can pick `/^00\d{8}$/,` from a constant since majority of the ID patterns will be same pattern pleas.

Reply:
Changed to use UKEFID.MAIN_ID.TEN_DIGIT_REGEX.